<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminWindow</class>
 <widget class="QMainWindow" name="adminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>40</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <widget class="QComboBox" name="funComboBox">
      <item>
       <property name="text">
        <string>员工管理</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>货物管理</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="goodsListButton">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>货物查询</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="delStaffButton">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>删除员工</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="addStaffButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>添加员工</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="retuenLoginButton">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QListWidget" name="staffListWidget">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="buyGoodsButton">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>货物采购</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>retuenLoginButton</zorder>
   <zorder>funComboBox</zorder>
   <zorder>goodsListButton</zorder>
   <zorder>delStaffButton</zorder>
   <zorder>addStaffButton</zorder>
   <zorder>staffListWidget</zorder>
   <zorder>buyGoodsButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>522</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
